### PR TITLE
Removing unicode from coset_table and fp_groups

### PR DESCRIPTION
**Pull request by sylee957**
#### References to other Issues or PRs

#16272

#### Brief description of what is fixed or changed

With the implementation in #16272, I could pinpoint the modules using unicode.
And I find many SymPy module already using lots of unicode for math notations or historic figures, or such.

I have partially done some cleanups, and am showcasing here at this point, because lots of things would be clear and repetitive after the decision made, and it is not yet fully decided that this work should be continued.

To summarize the previous discussions, we had reached the that the unicode charcters should be avoided in most cases, because of the different editors or fonts people are using.

But there may be some argument about this, like using ascii can make some formula verbose and less intuitive, or such.

Though my opinion, is pro about not using unicode in the generic codebase.
Because there is an alternative option to move such verbose comments, like full algorithm or math formula explanations, into the docstring as LaTeX.

#### Release Notes

NO ENTRY

### CODE REVIEW

**Comment by jksuom** [comment]
+1

**Comment by oscarbenjamin** [comment]
+1

I agree that using LaTeX where possible makes sense. Most examples here are in the comments though. If we were going to allow unicode in the source then the obvious big win would be the doctests. Using unicode in the comments just seems unnecessary if we're not going to use it where it really makes a difference.

File: sympy/combinatorics/coset_table.py [hunk 1]
```diff
@@ -949,7 +948,7 @@ def modified_coincidence(self, alpha, beta, w):
         Parameters
         ==========
 
-        A coincident pair \alpha,\beta \in \Omega, w \in (Y∪Y^–1)
+        A coincident pair \alpha,\beta \in \Omega, w \in `Y \cup Y^{-1}`
```
Comment (oscargus) [inline sympy/combinatorics/coset_table.py]: Shouldn't \alpha etc also be equations? (I know this is not what you have fixed though...)

**Comment by oscargus** [comment]
I think this makes sense as well. Especially since the code variables are not unicode characters.

One may also conceptually discuss if a variable alpha should be rendered as `\alpha` in the documentation. I think that it makes sense to do that.

**Comment by oscargus** [comment]
Do you know why your new commits does not cancel the old Travis builds automatically? I think it happens for (almost) everyone else, but not for you...

**Comment by sylee957** [comment]
@oscargus 
Well, I suppose that the travis cancels the job in 'stagewise', and if you have some jobs running already, it may not cancel even if a new commit is pushed.

Though I am not sure about my observations, and my information can be wrong.
But if we find that this is true, we may have to separate the third stage.

**Comment by oscargus** [comment]
To me it seems more like it is user dependent, but I really have no idea. I have just noted that sometimes it cancels when you push something new and sometimes not. I have not really collected information (as it is also hard to know if someone else has cancelled it), but I have recently noticed that for your commits it does not seem to happen.

Will collect a bit more information... (I cancelled them now, btw.)

**Comment by sylee957** [comment]
@oscargus 
I also have the travis on my fork,
https://travis-ci.org/sylee957/sympy
And, actually, It's clearly visible that the travis not canceling the job, if the already running the 3rd stage, right now.

**Comment by oscargus** [comment]
But, yes you may be correct that it is related to the stage thing, that it only cancels before starting a new stage. I usually commit (some fixes for stupid errors) quite rapidly after my first commit...

**Comment by sylee957** [comment]
I have opened up a discussion at #16335

**Pull request merged**